src/com/projectkorra/projectkorra/configuration/ConfigManager.java

```diff
@@ -73,9 +75,16 @@ public static void configCheck(final ConfigType type) {
 			config.addDefault("Chat.Prefixes.Nonbender", "[Nonbender]");
 
 			config.addDefault("Board.Title", "&lAbilities");
-			config.addDefault("Board.SelectionPrefix", ">  &r");
+			config.addDefault("Board.Prefix.Text", "> ");
+			config.addDefault("Board.Prefix.SelectedColor", ChatColor.WHITE.getName());
+			config.addDefault("Board.Prefix.NonSelectedColor", ChatColor.DARK_GRAY.getName());
 			config.addDefault("Board.EmptySlot", "&8-- Slot {slot_number} --");
-			config.addDefault("Board.MiscSeparator", "  ------------");
+			config.addDefault("Board.MiscSeparator", "  ----------");
+			
+			if (!config.contains("Board.Extras")) {
+				config.addDefault("Board.Extras.RaiseEarthWall", ChatColor.GREEN.getName());
+				config.addDefault("Board.Extras.SurgeWave", ChatColor.AQUA.getName());

```

Comment:
Why are we doing this only when the config doesn't contain it. I feel like this should always be added.

Reply:
I might be wrong but I think otherwise it would add this section even if the user doesn't want it.

Reply:
Configuration is designed so that all options are automatically available to you. This seems to contradict that purpose by selectively adding things to the config based on whether the user wants it or not.

Instead we should get rid of the conditional and add a third "Enabled" config option that users can toggle themselves.

Reply:
Since the name is part of the key in the config, like robbie says, then it'll always be added back even if the server owner deletes it. This way, it won't be added back unless they delete the entire Extras section.

Reply:
My whole point is why are we allowing users to delete parts of the configuration.

Reply:
it's setup this way so that the keys in the Extras section act as a list of extra names to look for when cooldowns are applied, and the value for the key is the ChatColor they want it to appear as. This way I don't have to setup some kind of parser.

Reply:
The purpose of this section is so people can add their own configuration sections here, so that they can write an ability name and, under it, specify the color. Having these two here is a good default but should not be obligatory, and adding an Enabled thing would be pointless if you can just remove the two config keys. We want people to be able to add their own moves so we don't have to check for everything.